### ISSUE TITLE
Horrible Rendering Issues

### ISSUE THREAD

**Issue by LegozForDayz** [mnurzia/mint-jams#1]

**Comment by LegozForDayz** [comment]
> When I try to download the poster, it does not render correctly and my screen flashes black and white back and forth (you cannot see the poster being rendered). The only way is to do Ctrl + P for print preview (to try to save to pdf) and even then, the front cover and the four faces of the band members don't appear. I'm on a laptop with a Ryzen 7 4700U so I don't know if I need something more powerful. When I try on my phone to download the poster, it shows the poster immediately but without the front cover and the band member's faces (as I said above). My pitiful attempt can be seen below [poster.pdf](https://github.com/mnurzia/mint-jams/files/7606202/poster.pdf) .

So I managed to fix the issues by opening it in Inkscape and then exporting as a png. I wonder how mnurzia did this, it's an amazing remaster/cleanup of Mint Jams.

**Comment by mnurzia** [comment]
@LegozForDayz, sorry I didn't catch this. I know you figured out a workaround, but I just pushed a fix that separates most of the source images from the .svgs. The original album art scan I used was like 8MB, and Inkscape didn't seem to like that. All the images I used to trace it are in the sources directory.

And to answer your question about how I traced it, I basically just spent an entire weekend in March 2020 tracing the shapes from the original art using Bezier curves. It was a long and tedious process, but it was definitely worth it. For the back, I matched up the font (Helvetica 😄) and then adjusted the kerning a bit.
